src/lang/en-CA.json

```diff
@@ -117,20 +119,33 @@
 	"error.invalid-request": "Invalid request",
 	"error.no-location-entered": "Please enter a city or state in the location search box above.",
 	"error.sign-in-to-share-resources": "Oops! You need to be signed in to share resources.",
+	"error.sign-in-to-delete-resources": "¡Oops! You need to be signed in to delete resources.",
 	"error.invalid-reset-token": "Invalid password reset token",
 	"error.required-field-empty": "Please fill out all fields",
 	"error.email-address-mismatch": "The email addresses you have entered do not match",
 	"error.page-not-found": "Page not found",
 	"error.form-description-length": "Description cannot be longer than 1000 characters",
-	"favorites.create-new-list": "Create New Favourites List",
+	"error.session-expired": "Your session has expired. Please log in again.",
+	"favorites.title": "Favourites",
 	"favorites.view-favorites": "View Your Favourites",
+	"favorites.no-lists": "You haven't created any lists yet.",
+	"favorites.create-new-list": "Create a New Favourites List",
 	"favorites.name-list-help": "Name your list by category, day of the week, or the name of whomever this list is for",
-	"favorites.privacy-disclaimer": "Your favourites lists are only visible to you and anyone you share them with.",
+	"favorites.privacy-disclaimer": "Your favourites lists are only visible to you and anyone you choose to share your lists with.",
```

Comment:
ok, I changed this because of what was in the localization link you sent.
<img width="993" alt="Screen Shot 2021-09-30 at 12 46 01 PM" src="https://user-images.githubusercontent.com/10123216/135520805-dafb5d39-a5ab-4e5d-b480-ebcb49b6aa7c.png">

Reply:
understood. however, the file was already marked as completed.

Reply:
Understood. the existing language is cleaner though

Reply:
@FJKhan ok, I will revert this and also update the localization link. 
Thanks for the feedback!